src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -191,7 +191,7 @@ private async ValueTask<FrameHeader> ReadFrameAsync()
             return frameHeader;
         }
 
-        private async void ProcessIncomingFrames()
+        private async Task ProcessIncomingFrames()
```

Comment:
nit: maybe this is a good time to rename this to "ProcessIncomingFramesAsync" ?

Reply:
Renamed.